services/common/src/@Types/global.d.ts

```diff
@@ -0,0 +1,13 @@
+declare global {
```

Comment:
This type is to make the REQUEST_HEADER available globally in the common package (it's mapped to each FE's REQUEST_HEADER through their individual webpack configs.